test/test-cases.txt

```diff
@@ -90,3 +90,9 @@
 # Text following a <br>
 {"blocks":[{"type":"unstyled","entityNodes":[{"entity":null,"styleNodes":[{"text":"Hello\nWorld!","styles":null}]}]}]}
 <p>Hello<br /> World!</p>
+
+# HTML video with iframe
+{"entityMap":{"0":{"type":"VIDEO","data":{"src":"https://www.youtube.com/embed/sFXQwk63Y9E?showinfo=0&rel=0", "width":"400", "height":"300"}}},"blocks":[{"type":"unstyled","entityNodes":[{"entity":"0","styleNodes":[{"text":"</p>","styles":null}]}]}]}
```

Comment:
@sstur hmm.. its still broken, but the difference is: before adding iframe to SELF_CLOSING ELEMENTS, the text was `text: "</div>"`, but after adding, the text is `text: "</div> ` (space added)